changelog/changelog_3.0.0-4.0.0.txt

```diff
@@ -1,11 +1,45 @@
+22.07.2024
+Description:
+	- Standardize cases to PascalCase
+		- Component IDs
+			- Reference Device
+			- Reference Function Blocks (backwards compatible)
+				- Classifier
+				- FFT
+				- Power
+				- Renderer
+				- Scaling
+				- Statistics
+				- Trigger
+			- AudioDeviceModuleWavWriter
+			- Reference device IO components (AI, CAN, RefCh)
+		- Component names
+			- Default client device
+		- Type IDs
+			- Reference modules
+			- Streaming/config clients (backwards compatible)
+			- Server modules
+			- MiniAudio
+		- Type names
+			- Reference modules
+			- Streaming/config clients
+			- Server modules
+		- Server capability protocol ID (backwards compatible)
+		- Server capability protocol name
+		- Struct Type field names
+		- Other
+
+Required integration changes:
+	- Generally none, except for where integration depends upon changed strings listed above (in the description) in some way
```

Comment:
I'd add some extra information:
- If relying on string comparison to hardcoded old IDs of things like FB, device, server types, or protocol  IDs, those comparisons will need to be updated to match the new IDs. eg. a check like `if (fbType.getId() == "ref_fb_module_renderer")` will never be true
- Old IDs can still be used when adding new objects to a device via `addDevice`/`addFunctionBlock` or similar calls.